app/controllers/resources_controller.rb

```diff
@@ -1,15 +1,16 @@
 class ResourcesController < ApplicationController
 
-  before_action :set_resource, only: [:show, :edit, :update, :destroy]
+  before_action :set_resource, only: [:show, :edit, :update, :like, :destroy]
 
   # GET /resources
   # GET /resources.json
   def index
     if params[:tag]
-      @resources = Resource.tagged_with(params[:tag])
+      @resources = Resource.tagged_with(params[:tag]).order(date: :desc, created_at: :desc)
     else
-      @resources = Resource.all
+      @resources = Resource.order(date: :desc, created_at: :desc)
     end
```

Comment:
Could you filter this further and only show those resources that are marked as 'public'?